mmv1/products/compute/api.yaml

```diff
@@ -8055,6 +8055,21 @@ objects:
           Maximum Transmission Unit in bytes. The minimum value for this field is 1460
           and the maximum value is 1500 bytes.
         input: true
+      - !ruby/object:Api::Type::Boolean
+        name: 'enableUlaInternalIpv6'
+        description: |
+          Enable ULA internal ipv6 on this network. Enabling this feature will assign 
+          a /48 from google defined ULA prefix fd20::/20.
+        input: true
+      - !ruby/object:Api::Type::String
+        name: 'internalIpv6Range'
```

Comment:
you'll need to make this field `default_from_api: true` in the `terraform.yaml`.

The test is failing because it doesn't know this value can be inferred.

Reply:
Done